util/src/main/i18n/templates/error.properties

```diff
@@ -7,4 +7,14 @@ error.invalidFormat = Invalid format: '{0}' is not a {1}
 error.outOfRange = Invalid value: {0} is not between {1}
 error.suggestionSuffix = (did you mean '{0}'?)
 
+# {0} - the error message
+error.config = Config error: {0}
```

Comment:
This is too generic of a message. Error messages should be friendly and understandable by users that don't know anything about the codebase.

Reply:
The {0} in Config error is the actual error.

Reply:
So, it would be `Config Error: Bad credentials: {0} username or password is incorrect`